Dockerfile

```diff
@@ -1,14 +1,12 @@
-# latest kubectl in the SDK
-# https://cloud.google.com/sdk/docs/release-notes
-FROM google/cloud-sdk:alpine
-
-# Install kubectl
-RUN gcloud components install kubectl && \
-    rm -rf ./google-cloud-sdk/.install
+# see https://hub.docker.com/r/google/cloud-sdk/tags for available tags / versions
+ARG GCLOUD_SDK_VERSION
 
+FROM google/cloud-sdk:${GCLOUD_SDK_VERSION} AS cloud-sdk
 ENV CLOUDSDK_CONTAINER_USE_APPLICATION_DEFAULT_CREDENTIALS=true
+ENV CLOUDSDK_CORE_DISABLE_PROMPTS=1
+ADD bin/install-kubectl /usr/local/bin/
+RUN install-kubectl && rm -f /usr/local/bin/install-kubectl
 
-# Add the Drone plugin
-ADD drone-gke /bin/
-
-ENTRYPOINT ["/bin/drone-gke"]
+FROM cloud-sdk
```

Comment:
Any reason for splitting this up like that instead of just having ADD and ENTRYPOINT?

Reply:
@fsouza probably not at this point, no. i had started down a different path w/ this implementation originally .. creating that `cloud-sdk` as a completely separate base image then doing something like:

```dockerfile
FROM nytimes/gcloud-sdk-kubectl:1.14
# ...
```

while probably more generally useful (since that image could be used in other cases), it would have added complexity here, which I was trying to limit.

that said, happy to use a non multi-stage build here if you'd prefer!

Reply:
Yeah I think having a single stage would make it more straightforward for the reader, we can always change in the future! :D